src/library/scala/util/Either.scala

```diff
@@ -168,7 +168,7 @@ sealed abstract class Either[+A, +B] extends Product with Serializable {
    *
    *  Because `Either` is right-biased, this method is not normally needed.
    */
-  @deprecated("Either is now right-biased", "2.13.0")
+  @deprecated("Either is now right-biased, use methods directly on Either", "2.13.0")
```

Comment:
Sounds reasonable.

@SethTisue can and should we still squeeze that in?

Reply:
I added it to the PR, but I have no strong feelings on the matter one way or the other.

Reply:
meh, I think I'd rather not add more unsafe, non-total methods unless it's really a clear win. admittedly there is precedent on `Option`, but I basically think nobody's asking for this and we're probably better off without it. just as we might arguably be better off without `Option#get` (which we're probably stuck with, because precedent)

`.getOrElse(throw ...)` is always there

Reply:
Then we need a better deprecation message when calling .right.get, I guess people are likely to replace it with .toOption.get (which is why I'd rather avoid the indirection and have Either#get)

Reply:
Actually .toOption.get isn't great because it won't preserve the value of Left in the exception shown (which could be critical for debugging)

Reply:
I pushed a new proposal to this branch: the deprecated message points to `getOrElse`, and `get` is removed.